mautic/mautic
# Fixes #1798, where user couldn't re-subscribe contact manually from contact details page

Please answer the following questions:

| Q             | A
| ------------- | ---
| Bug fix?      | Y
| New feature?  | 
| BC breaks?    | 
| Deprecations? | 
| Fixed issues  |  #1798

## Description

See #1798

## Steps to reproduce the bug (if applicable)

- Go to Contacts
- Unsubscribe manually a contact
- Go to details of contact
- Click on little cross to clear 'Do Not Contact' entry (re-subscribe)
- When on prod env (index.php) nothing happens and keeps loading forever and when on dev env (index_dev.php) an error is shown

## Steps to test this PR

- Apply this PR
- Re-test, refresh the page and click again on the little cross. Now you should be able to clear "Do Not Contact" entry
